### DISCUSSION TITLE
Testing in 2024

### DISCUSSION THREAD

**Discussion by Entity-sudo** [Discord-OTP-Forcer/Discord-OTP-Forcer#105] [General]
Started to use this program, currently using reset with only 6 digit attempts will try and let run for as long as i can,
but i am wondering how long with this theoretically take, and is it just pure luck? Does the otp change every 30 seconds? 
Some side information.
for those like me who arnt especially skilled in python, with the files, the startup.bat need to be ran while the whole file is still zipped, then extract the whole file when starting it.
i also ran into issues where windows didn't, or couldn't find python even though it was installed, but demanded to be installed through the Microsoft store for it to work.
    Currently at just over 550 attempts with no need for a new token, or refresh or anything. just keeps going.
will come back and update if i succeed. 
hopefully i succeed

**Comment by TheRealRedeye** [comment]
> For those like me who arnt especially skilled in python, with the files, the startup.bat need to be ran while the whole file is still zipped, then extract the whole file when starting it.

I've used this program before but started running into this issue after a factory reset and reinstall of everything, the way you worded this has me confused though, I'm unable to save any of my information into the setup.bat while it's still zipped so how could I run it in the first place?

**Reply by Entity-sudo** [reply]
i apologies for my bad explaining. hope the other way works better for you. if still an issue i can replicate it if need be
